frameworks/digital-outcomes-and-specialists/questions/brief-responses/newAvailabilityOutcomes.yml

```diff
@@ -0,0 +1,19 @@
+name: Earliest start date
+id: availability
+question: When is the earliest the team can start
```

Comment:
I think this text is the only difference between two questions. We can use the Jinja `{% if lot == ... %}` here and merge them together